Library/Homebrew/livecheck/strategy.rb

```diff
@@ -181,18 +181,25 @@ def from_url(url, livecheck_strategy: nil, url_provided: false, regex_provided:
       # collected into an array of hashes.
       #
       # @param url [String] the URL to fetch
-      # @param homebrew_curl [Boolean] whether to use brewed curl with the URL
+      # @param options [Hash] options to control behavior
       # @return [Array]
-      sig { params(url: String, homebrew_curl: T::Boolean).returns(T::Array[T::Hash[String, String]]) }
-      def self.page_headers(url, homebrew_curl: false)
+      sig { params(url: String, options: T::Hash[Symbol, T.untyped]).returns(T::Array[T::Hash[String, String]]) }
+      def self.page_headers(url, options = {})
```

Comment:
Let's make these named keyword parameters rather than an options hash so they can be properly typed and checked.